packages/core/src/components/axes/zoom-bar.ts

```diff
@@ -43,8 +48,12 @@ export class ZoomBar extends Component {
 	render(animate = true) {
 		const svg = this.getContainerSVG();
 
-		const isTopZoomBarLoading = this.services.zoom.isZoomBarLoading(AxisPositions.TOP);
-		const isTopZoomBarLocked = this.services.zoom.isZoomBarLocked(AxisPositions.TOP);
+		const isTopZoomBarLoading = this.services.zoom.isZoomBarLoading(
+			AxisPositions.TOP
+		);
+		const isTopZoomBarLocked = this.services.zoom.isZoomBarLocked(
+			AxisPositions.TOP
+		);
```

Comment:
Why can't these stay on their own lines?
```suggestion
		const isTopZoomBarLoading = this.services.zoom.isZoomBarLoading(AxisPositions.TOP);
		const isTopZoomBarLocked = this.services.zoom.isZoomBarLocked(AxisPositions.TOP);
```

Reply:
Prettier will format these anyways. Looks like it decided to do that

Reply:
That's right. If we like to keep them in singe line, we need to change the prettier configuration.